src/ontology/uberon-edit.obo

```diff
@@ -224701,6 +224701,85 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
 intersection_of: connects UBERON:7500048 ! proximal-most point of ventral tubercle of humerus
 created_by: https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500063
+name: trochlea of talus
+def: "The articular surface of a talus that interacts with a tibia. It has a deep depression that forms the trochlear groove and is bounded by the medial and lateral condyles." [https://en.wikipedia.org/wiki/Talus_bone]
+synonym: "trochlea for tibia" EXACT []
+synonym: "trochlea of astragalus" EXACT []
+synonym: "trochlea tali" EXACT []
+is_a: UBERON:0004120 ! mesoderm-derived structure
+relationship: part_of UBERON:0002395 ! talus
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T00:56:19Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500064
+name: trochlear groove of talus
+def: "The furrow between the lateral and medial condyles of the trochlea of talus." [https://en.wikipedia.org/wiki/Talus_bone]
+synonym: "trochlear groove of astragalus" EXACT []
+is_a: UBERON:0000064 ! organ part
+is_a: UBERON:0004120 ! mesoderm-derived structure
+relationship: part_of UBERON:0002395 ! talus
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T00:57:49Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500065
+name: condyle of talus
+def: "An articular prominence of a talus that engages with the tibia." []
+synonym: "condyle of astragalus" EXACT []
+intersection_of: UBERON:0009979 ! condyle
+intersection_of: part_of UBERON:0002395 ! talus
+property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T00:59:42Z xsd:dateTime
+property_value: term:editor http://orcid.org/0000-0003-2699-3066
+
+[Term]
+id: UBERON:7500066
+name: medial condyle of talus
+def: "The medial condyle of talus is an articular prominence of bone on medial side of the trochlea of a talus, which engages with the tibia." [ORCID: 0000-0003-2699-3066]
+synonym: "medial condyle of astragalus" EXACT []
+is_a: UBERON:7500065 ! condyle of talus
```

Comment:
you can let the reasoner infer this from 
is_a: condyle
relationship: part_of talus

Dynamic is generally better.

Reply:
done.